src/System.IO.FileSystem.AccessControl/dir.props

```diff
@@ -3,5 +3,7 @@
   <Import Project="..\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.2.0</AssemblyVersion>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
It's an implementation dependency of IsolatedStorage:
https://github.com/dotnet/corefx/blob/8590e419f6639d9d5eabdbcb916e62fb1656d30e/src/System.IO.IsolatedStorage/src/System.IO.IsolatedStorage.csproj#L49

@JeremyKuhne added the dependency here: https://github.com/dotnet/corefx/commit/86f554277b6859fe413948b15c81fd34e1288140